packages/libnvidia-container/libnvidia-container.spec

```
@@ -0,0 +1,76 @@
+%global nvidia_modprobe_version 450.57
+
+Name: %{_cross_os}libnvidia-container
+Version: 1.5.0
+Release: 1%{?dist}
+Summary: NVIDIA container runtime library
+License: Apache-2.0 and GPL-2.0-only
+URL: https://github.com/NVIDIA/libnvidia-container
+Source0: https://github.com/NVIDIA/libnvidia-container/archive/v%{version}.tar.gz
+Source1: https://github.com/NVIDIA/nvidia-modprobe/archive/%{nvidia_modprobe_version}/nvidia-modprobe-%{nvidia_modprobe_version}.tar.gz
+
+# First party patches from 1 to 1000
+Patch0001: 0001-use-shared-libtirpc.patch
+Patch0002: 0002-use-prefix-from-environment.patch
+Patch0003: 0003-keep-debug-symbols.patch
+Patch0004: 0004-Use-NVIDIA_PATH-to-look-up-binaries.patch
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libelf-devel
+BuildRequires: %{_cross_os}libcap-devel
+BuildRequires: %{_cross_os}libseccomp-devel
+BuildRequires: %{_cross_os}libtirpc-devel
+Requires: %{_cross_os}libelf
+Requires: %{_cross_os}libcap
+Requires: %{_cross_os}libseccomp
+Requires: %{_cross_os}libtirpc
+
+%description
+%{summary}.
+
+%package devel
+Summary: NVIDIA contanier runtime library development files
```

typo in `container`, but also our standard `devel` summary is "Files for development using the [main package summary]".

```suggestion
Summary: Files for development using the NVIDIA container runtime library
```